akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -1060,39 +1064,14 @@ class Http2ServerSpec extends AkkaSpecWithMaterializer("""
 
     def sendBytes(bytes: ByteString): Unit = fromNet.sendNext(bytes)
 
-    def sendFrame(frame: FrameEvent): Unit =
-      sendBytes(FrameRenderer.render(frame))
-
-    def sendFrame(frameType: FrameType, flags: ByteFlag, streamId: Int, payload: ByteString): Unit =
-      sendBytes(FrameRenderer.renderFrame(frameType, flags, streamId, payload))
-
-    def sendDATA(streamId: Int, endStream: Boolean, data: ByteString): Unit = {
+    override def sendDATA(streamId: Int, endStream: Boolean, data: ByteString): Unit = {
       updateToServerWindowForConnection(_ - data.length)
       updateToServerWindows(streamId, _ - data.length)
-      sendFrame(FrameType.DATA, Flags.END_STREAM.ifSet(endStream), streamId, data)
-    }
-
-    def sendSETTING(identifier: SettingIdentifier, value: Int): Unit =
-      sendFrame(SettingsFrame(Setting(identifier, value) :: Nil))
-
-    def sendHEADERS(streamId: Int, endStream: Boolean, endHeaders: Boolean, headerBlockFragment: ByteString): Unit =
-      sendBytes(FrameRenderer.render(HeadersFrame(streamId, endStream, endHeaders, headerBlockFragment, None)))
-
-    def sendCONTINUATION(streamId: Int, endHeaders: Boolean, headerBlockFragment: ByteString): Unit =
-      sendBytes(FrameRenderer.render(ContinuationFrame(streamId, endHeaders, headerBlockFragment)))
-
-    def sendPRIORITY(streamId: Int, exclusiveFlag: Boolean, streamDependency: Int, weight: Int): Unit =
-      sendBytes(FrameRenderer.render(PriorityFrame(streamId, exclusiveFlag, streamDependency, weight)))
-
-    def sendRST_STREAM(streamId: Int, errorCode: ErrorCode): Unit = {
-      implicit val bigEndian: ByteOrder = ByteOrder.BIG_ENDIAN
-      val bb = new ByteStringBuilder
-      bb.putInt(errorCode.id)
-      sendFrame(FrameType.RST_STREAM, ByteFlag.Zero, streamId, bb.result())
+      super.sendDATA(streamId, endStream, data)
     }
 
-    def sendWINDOW_UPDATE(streamId: Int, windowSizeIncrement: Int): Unit = {
-      sendBytes(FrameRenderer.render(WindowUpdateFrame(streamId, windowSizeIncrement)))
+    override def sendWINDOW_UPDATE(streamId: Int, windowSizeIncrement: Int): Unit = {
```

Comment:
The window stuff is the same on client and server side so it might make sense to extract that as well. But this can maybe be done in another PR.